README.md

```diff
@@ -35,6 +35,12 @@ just deploy-astria-local
 
 # Deploys a geth rollup chain + faucet + blockscout + ingress
 just deploy-rollup
```

Comment:
Should we remove the deploy-rollup and other geth deployment options, rename `deploy-rollup-chart` to `deploy-rollup`.

This can be follow up, but this seems to be code we ideally don't use anymore. We could add defaults for the chart command as well.